docs/modding/INSTALLING_PACKAGES.md

```
@@ -0,0 +1,94 @@
+
+## Inspecting a Package Before Installing
+
+Before installing an unfamiliar `.ofm` file, use `ofm-cli info` to see its contents:
+
+```bash
+ofm-cli info some-package.ofm
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)